include/astshim/FrameSet.h

```diff
@@ -409,7 +409,10 @@ friend class Object;
     };
 
     /**
-    Remove a @ref Frame Frame from a @ref FrameSet
+    Remove a @ref Frame Frame from a @ref FrameSet.
```

Comment:
Thanks for spotting this. I changed `@ref Frame Frame` to `@ref Frame` in the only two places I found it. I would dearly love to get rid of all unnecessary `@ref` tags, which I think means all those  that do not specify alternate text for the link, such as `@ref Frame "Frames"`. However, I have not thought of a suitable grep expression or script. Suggestions welcome.